cmd/plugins/juju-metadata/toolsmetadata.go

```diff
@@ -61,21 +62,16 @@ use the --clean option.
 Examples:
 
   - generate metadata for "released" tools, looking in the "releases" directory:
-
-   juju metadata generate-tools -d <workingdir>
+    juju metadata generate-tools -d <workingdir>
 
   - generate metadata for "released" tools, looking in the "released" directory:
```

Comment:
Not your change I know but can we get rid of the word "directory" in these examples. It's confusing.

It would be enough to just say:
```
- generate metadata for "released" tools
```

Also, in other help examples, the comments tend to start with `#`. Can you please make this consistent while you're in here?